stubs/django-filter/django_filters/__init__.pyi

```diff
@@ -0,0 +1,9 @@
+from typing import Any
+
+from . import rest_framework as rest_framework
+from .filters import *
+from .filterset import FilterSet as FilterSet
+
+def parse_version(version: Any): ...
```

Comment:
We now believe Any should be used only in places where the type system can't express the precise type. As a result, we'd want a comment with every Any explaining why it's there.

If this is Any merely because you haven't figured out the precise type, you can leave the parameter unannotated, or use `_typeshed.Incomplete` in cases where that's not possible (e.g., for the global below).